BREAKING.md

```diff
@@ -22,11 +22,16 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 ## 2.0.0-internal.4.0.0 Breaking changes
 
 -   [Container and RelativeLoader no longer exported](#Container-and-RelativeLoader-no-longer-exported)
+-   [IFluidHTMLView, ReactViewAdapter, and HTMLViewAdapter removed](#IFluidHTMLView-ReactViewAdapter-and-HTMLViewAdapter-removed)
 
 ### Container and RelativeLoader no longer exported
 
 Container and RelativeLoader are no longer exported. All Container usages should have previously moved to IContainer. RelativeLoader is an internal implementation which should not be exposed or used directly.
 
+### IFluidHTMLView, ReactViewAdapter, and HTMLViewAdapter removed
+
+`IFluidHTMLView`, `ReactViewAdapter`, and `HTMLViewAdapter` were deprecated in a prior release, and are now removed.
```

Comment:
Maybe just call out the specific release where we removed these APIs, so it's clear if it's minor/major etc. (As a general rule we try to aim majors, so its good to make the "when" visible here)

Reply:
Done in latest